aviary/interface/methods_for_level2.py

```diff
@@ -136,8 +136,8 @@ def configure(self):
         # Find promoted name of every input in the model.
         all_prom_inputs = []
 
-        # We can call list_inputs on the groups.
-        for system in self.system_iter(recurse=False, typ=om.Group):
```

Comment:
Do you remember why you needed to iterate on components here? We iterated on them in the next block of code. Components aren't quite finished setting up at this point because you are still allowed to add input so them from their owning group during configure.  Note, I am having some different issues with these list_inputs calls, so am interested in your data.

Reply:
this was from your PR https://github.com/crecine/om-Aviary/pull/28/files
I think it was related to setting overrides for variables calculated in pre-mission that aren't used until the mission or post-mission

Reply:
Oh, I guess this fix never made it to main because I did it directly on your branch.